tests/test_fips.py

```diff
@@ -60,7 +58,7 @@ def test_enable_fips_already_enabled(self):
 
     def test_enable_fips_installed_not_enabled(self):
         """If fips is installed but not enabled an error is returned."""
-        self.make_fake_binary('dpkg-query')
+        self.setup_fips()
```

Comment:
We may not gain much, but I would think it more logical to call setup_fips() in the test setUp() with some reasonable default. ESM and livepatch already do this.
I counted the setup_fips() calls in the existing tests and we call it about 3 times to enable fips, and 3 times to disable it, so whatever our default state should be is a 50-50 chance of being correct most of the time :)

Reply:
Yeah but we only want to call `setup_fips()` if there's something to set up. The FIPS file must not always be there, and always mocking `dpkg-query` migth cover failures